security-jwt/src/main/java/io/micronaut/security/token/jwt/cookie/JwtCookieClearerLogoutHandler.java

```diff
@@ -39,46 +40,76 @@
 @Singleton
 public class JwtCookieClearerLogoutHandler implements LogoutHandler {
 
+    @Deprecated
     protected final JwtCookieConfiguration jwtCookieConfiguration;
     protected final String logout;
+    protected final AccessTokenCookieConfiguration accessTokenCookieConfiguration;
+    protected final RefreshTokenCookieConfiguration refreshTokenCookieConfiguration;
 
     /**
      * @param jwtCookieConfiguration JWT Cookie Configuration
      * @deprecated Use {@link JwtCookieClearerLogoutHandler#JwtCookieClearerLogoutHandler(JwtCookieConfiguration, RedirectConfiguration)} instead.
      */
     @Deprecated
     public JwtCookieClearerLogoutHandler(JwtCookieConfiguration jwtCookieConfiguration) {
-            this.jwtCookieConfiguration = jwtCookieConfiguration;
-            this.logout = jwtCookieConfiguration.getLogoutTargetUrl();
+        this.jwtCookieConfiguration = jwtCookieConfiguration;
+        this.accessTokenCookieConfiguration = jwtCookieConfiguration;
+        this.refreshTokenCookieConfiguration = null;
+        this.logout = jwtCookieConfiguration.getLogoutTargetUrl();
     }
 
     /**
      * @param jwtCookieConfiguration JWT Cookie Configuration
      * @param redirectConfiguration Redirect configuration
      */
-    @Inject
+    @Deprecated
     public JwtCookieClearerLogoutHandler(JwtCookieConfiguration jwtCookieConfiguration,
                                          RedirectConfiguration redirectConfiguration) {
         this.jwtCookieConfiguration = jwtCookieConfiguration;
+        this.accessTokenCookieConfiguration = jwtCookieConfiguration;
+        this.refreshTokenCookieConfiguration = null;
+        this.logout = redirectConfiguration.getLogout();
+    }
+
+    /**
+     * @param accessTokenCookieConfiguration JWT Cookie Configuration
+     * @param refreshTokenCookieConfiguration Refresh token cookie configuration
+     * @param redirectConfiguration Redirect configuration
+     */
+    @Inject
+    public JwtCookieClearerLogoutHandler(AccessTokenCookieConfiguration accessTokenCookieConfiguration,
+                                         RefreshTokenCookieConfiguration refreshTokenCookieConfiguration,
+                                         RedirectConfiguration redirectConfiguration) {
+        this.accessTokenCookieConfiguration = accessTokenCookieConfiguration;
+        this.refreshTokenCookieConfiguration = refreshTokenCookieConfiguration;
+        this.jwtCookieConfiguration = null;
```

Comment:
since `jwtCookieConfiguration`  had protected visibility. If anyone was overriding this class it would be better to set it 

`this.jwtCookieConfiguration = accessTokenCookieConfiguration;

instead of: 

`this.jwtCookieConfiguration = null;`